kodi_addon_checker/check_addon.py

```diff
@@ -413,8 +413,18 @@ def _get_users_dependencies(parsed_xml):
 def _check_dependencies(report: Report, repo_addons, parsed_xml):
     """Check for any new dependencies in addon.xml file"""
     deps = _get_users_dependencies(parsed_xml)
-    ignore = ['xbmc.json', 'xbmc.gui', 'xbmc.json',
-              'xbmc.metadata', 'xbmc.python']
+    ignore = ['xbmc.json', 'xbmc.gui', 'xbmc.metadata', 'xbmc.python', "xbmc.metadata.scraper.albums",
```

Comment:
your mixing `"` and `'` here. Settle for one :)
I would go with `'` I think.

And maybe even drop the old ones, as they are not in the reference list